synapse/module_api/__init__.py

```diff
@@ -1476,6 +1476,57 @@ async def lookup_room_alias(self, room_alias: str) -> Tuple[str, List[str]]:
 
         return room_id.to_string(), hosts
 
+    async def create_room(
+        self,
+        user_id: str,
+        config: JsonDict,
+        ratelimit: bool = True,
+        creator_join_profile: Optional[JsonDict] = None,
+    ) -> Tuple[str, Optional[str]]:
+        """Creates a new room.
+
+        Added in Synapse v1.65.0.
+
+        Args:
+            user_id:
+                The user who requested the room creation.
+            config : A dict of configuration options. See "Request body" of:
+                https://spec.matrix.org/latest/client-server-api/#post_matrixclientv3createroom
+            ratelimit: set to False to disable the rate limiter for this specific join.
```

Comment:
```suggestion
            ratelimit: set to False to disable the rate limiter for this specific operation.
```

Bleh, got confused with myself, apologies.